src/components/BuildConfigCreateEditPage/BuildConfigCreateEditPage.tsx

```diff
@@ -753,62 +867,64 @@ export const BuildConfigCreateEditPage = ({ isEditPage = false }: IBuildConfigCr
-                    onClick={() => {
-                      toggleCancelAllModal();
-                    }}
-                    isDisabled={!buildConfigChanges.length}
-                  >
-                    Cancel all
-                  </Button>
-                </ToolbarItem>
-              </Toolbar>
-              <ConfigsChangesList<BuildConfiguration>
-                variant="Build"
-                configChanges={buildConfigChanges}
-                onCancel={(buildConfig: BuildConfiguration) => {
-                  cancelBuildConfigChange(buildConfig);
-                }}
-              />
-            </GridItem>
-          </Grid>
-        </ExpandableSection>
-      </ContentBox>
+      {!isEditPage && (
+        <ContentBox marginBottom background={false} shadow={false}>
+          <ExpandableSection
+            title="Dependencies"
+            isExpanded={showDependenciesSection}
+            onToggle={(isExpanded) => setShowDependenciesSection(isExpanded)}
+          >
+            <Grid hasGutter>
+              <GridItem lg={12} xl2={6}>
+                <Toolbar borderBottom>
+                  <ToolbarItem>
+                    <TextContent>
+                      <Text component="h2">Add Build Config dependencies</Text>
+                    </TextContent>
+                  </ToolbarItem>
+                </Toolbar>
+                <ConfigsAddList<BuildConfiguration>
+                  variant="Build"
+                  serviceContainerConfigs={serviceContainerProjectBuildConfigs}
+                  componentId={componentIdBuildConfigs}
+                  onConfigAdd={(buildConfig: BuildConfiguration) => {
+                    insertBuildConfigChange(buildConfig, 'add');
+                  }}
+                  addedConfigs={addedBuildConfigs}
+                />
+              </GridItem>
+
+              <GridItem lg={12} xl2={6}>
+                <Toolbar>
+                  <ToolbarItem>
+                    <TextContent>
+                      <Text component="h2">Dependencies to be added</Text>
+                    </TextContent>
+                  </ToolbarItem>
+                  <ToolbarItem>
+                    <Button
+                      variant="tertiary"
+                      onClick={() => {
+                        toggleCancelAllModal();
+                      }}
+                      isDisabled={!buildConfigChanges.length}
+                    >
+                      Cancel all
+                    </Button>
+                  </ToolbarItem>
+                </Toolbar>
+                <ConfigsChangesList<BuildConfiguration>
+                  variant="Build"
+                  configChanges={buildConfigChanges}
+                  onCancel={(buildConfig: BuildConfiguration) => {
+                    cancelBuildConfigChange(buildConfig);
+                  }}
+                />
+              </GridItem>
+            </Grid>
+          </ExpandableSection>
+        </ContentBox>
+      )}
```

Comment:
Line 935: wrong submit button naming, see for example `ProjectCreateEditPage.tsx`